bundles/org.openhab.binding.miio/README.md

```diff
@@ -646,6 +646,13 @@ Additionally depending on the capabilities of your robot vacuum other channels m
 | Switch  | status#mop_forbidden_enable       | Mop Forbidden              |
 | Switch  | status#is_locating                | Robot is locating          |
 | Number  | actions#segment                   | Room Clean  (enter room #) |
+| Switch  | actions#collect_dust              | Start collect the dust at  |
+| Switch  | actions#clean_mop_star            | Start mop wash             |
+| Switch  | actions#clean_mop_stop            | Stop mop wash              |
+| Number  | status#mop_drying_time            | Mop Drying Time            |
+| Switch  | status#is_mop_drying              | Mop cleaning active        |
+
```

Comment:
Did you forget to add dock status channels here?

Reply:
Yeah i missed to run the Readmehelper,.. :-(